website/src/content/deployment-examples/azure.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Deploying to Microsoft Azure with Static Web Apps
```

Comment:
I would also rename the file to `azure-static-web-apps`, similarly to `aws-s3-cloudfront`

```suggestion
title: Deploying to Azure with Static Web Apps
```